tests_new/integration_tests/modules/share_base/fresh_deployment.py

```diff
@@ -0,0 +1,106 @@
+from assertpy import assert_that
+
+from dataall.modules.shares_base.services.shares_enums import ShareItemStatus, ShareObjectStatus
+from tests_new.integration_tests.errors import GqlError
+
+from tests_new.integration_tests.modules.share_base.queries import (
+    submit_share_object,
+    add_share_item,
+    get_share_object,
+    reject_share_object,
+    approve_share_object,
+    revoke_share_items,
+)
+from tests_new.integration_tests.modules.share_base.utils import check_share_ready
+
+
+def test_create_share_object(share1):
+    assert_that(share1.status).is_equal_to(ShareObjectStatus.Draft.value)
+
+
+def test_submit_empty_object(client2, share1):
+    # here Exception is not recognized as GqlError, so we use base class
+    # toDo: back to GqlError
+    assert_that(submit_share_object).raises(Exception).when_called_with(client2, share1.shareUri).contains(
+        'ShareItemsFound', 'The request is empty'
+    )
+
+
+def test_add_share_items(client2, share1):
+    items = share1['items'].nodes
+    assert_that(items[0].status).is_none()
+    assert_that(items).is_length(4)
+
+    item_to_add = items[0]
+    share_item_uri = add_share_item(client2, share1.shareUri, item_to_add.itemUri, item_to_add.itemType)
+    assert_that(share_item_uri).is_not_none()
+
+    updated_share = get_share_object(client2, share1.shareUri, {'isShared': True})
+    items = updated_share['items'].nodes
+    assert_that(items).is_length(1)
+    assert_that(items[0].shareItemUri).is_equal_to(share_item_uri)
+    assert_that(items[0].status).is_equal_to(ShareItemStatus.PendingApproval.value)
+
```

Comment:
My main concern is that we cannot test `test_reject_share` without running `test_submit_object_no_auto_approval` before. The tests are coupled, which in this case it might be alright, we are testing test_share_workflow succeed in parts. I would just clearly indicate that as a comment and group them all together (all tests on share1 together and all tests on share3 together to show the path). @petrkalos wdyt?